pkg/tests/apis/identity/identity_test.go

```diff
@@ -0,0 +1,150 @@
+package identity
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/grafana/grafana/pkg/services/featuremgmt"
+	"github.com/grafana/grafana/pkg/tests/apis"
+	"github.com/grafana/grafana/pkg/tests/testinfra"
+	"github.com/grafana/grafana/pkg/tests/testsuite"
+	"github.com/stretchr/testify/require"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+var gvrTeams = schema.GroupVersionResource{
+	Group:    "identity.grafana.app",
+	Version:  "v0alpha1",
+	Resource: "teams",
+}
+
+var gvrUsers = schema.GroupVersionResource{
+	Group:    "identity.grafana.app",
+	Version:  "v0alpha1",
+	Resource: "users",
+}
+
+func TestMain(m *testing.M) {
+	testsuite.Run(m)
+}
+
+func TestIntegrationRequiresDevMode(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping integration test")
+	}
+	helper := apis.NewK8sTestHelper(t, testinfra.GrafanaOpts{
+		AppModeProduction: true, // should fail
+		DisableAnonymous:  true,
+		EnableFeatureToggles: []string{
+			featuremgmt.FlagGrafanaAPIServerWithExperimentalAPIs, // Required to start the example service
+		},
+	})
+
+	_, err := helper.NewDiscoveryClient().ServerResourcesForGroupVersion("identity.grafana.app/v0alpha1")
+	require.Error(t, err)
+}
+
+func TestIntegrationIdentity(t *testing.T) {
```

Comment:
This integration test makes sure the SQL works in all three flavors MySQL/PostgrSQL/SQLite